guizero/LabelBox.py

```diff
@@ -0,0 +1,114 @@
+from tkinter import LabelFrame
+from . import utilities as utils
+from .base import ContainerTextWidget
+from .event import EventManager
+
+class LabelBox(ContainerTextWidget):
```

Comment:
Should `LabelBox` inherit from `Box`? `border`, `set_border` and `resize` methods are the same.